wgpu/gui/qt.py

```diff
@@ -244,12 +247,27 @@ def _key_event(self, event_type, event):
         }
         self._handle_event_and_flush(ev)
 
+    def _char_input_event(self, char, char_str):
+        ev = {
+            "event_type": "char",
+            "char": char,
+            "char_str": char_str,
+            "modifiers": None,
+        }
+        self._handle_event_and_flush(ev)
```

Comment:
A new event is introduced here. For the sake of portability, we should add it to jupyter_rfb (and its spec) too. I'm not sure if an event like this exists in JS. The `input` even comes close.

How badly is this new needed? Can we omit it for now, so we have time to investigate proper compat?

Or we we make it a private event for the time being ...

Reply:
> A new event is introduced here. For the sake of portability, we should add it to jupyter_rfb (and its spec) too. I'm not sure if an event like this exists in JS. The `input` even comes close.
> 
> How badly is this new needed? Can we omit it for now, so we have time to investigate proper compat?
> 
> Or we we make it a private event for the time being ...

It primarily handles character input events, which are somewhat different from key events. It is mainly used for UI controls that accept character input. Most importantly, it allows obtaining results from the input method, which is useful for inputting non-English characters.

Like this:
![image](https://github.com/pygfx/wgpu-py/assets/8044566/ff88bbd2-f466-4368-a773-a3971ae712b0)

Reply:
Ok, let's go with what we have.